request-server_test.go

```diff
@@ -590,6 +604,17 @@ func TestRequestSymlink(t *testing.T) {
 	require.NoError(t, err)
 	assert.Equal(t, []byte("hello"), content)
 
+	fi, err = r.lfetch("/link-to-non-existent-file")
+	require.NoError(t, err)
+	assert.True(t, fi.Mode()&os.ModeSymlink == os.ModeSymlink)
+
+	_, err = r.fetch("/link-to-non-existent-file")
+	require.True(t, os.IsNotExist(err))
```

Comment:
Same here, after adding the explanatory details we’re not getting anything out of the `require` package:
```
if !os.IsNotExist(err) {
  t.Fatal("expected file to not exist, but it did")
}
```